editor/import/resource_importer_wav.h

```diff
@@ -129,36 +118,25 @@ class ResourceImporterWAV : public ResourceImporter {
 
 				step >>= 1;
 				mask >>= 1;
-			};
+			}
 
 			if (nibble & 8) {
 				prev -= vpdiff;
 			} else {
 				prev += vpdiff;
 			}
 
-			if (prev > 32767) {
-				//printf("%i,xms %i, prev %i,diff %i, vpdiff %i, clip up %i\n",i,xm_sample,prev,diff,vpdiff,prev);
-				prev = 32767;
-			} else if (prev < -32768) {
-				//printf("%i,xms %i, prev %i,diff %i, vpdiff %i, clip down %i\n",i,xm_sample,prev,diff,vpdiff,prev);
-				prev = -32768;
-			}
+			prev = CLAMP(prev, -32768, 32767);
 
 			step_idx += _ima_adpcm_index_table[nibble];
-			if (step_idx < 0) {
-				step_idx = 0;
-			} else if (step_idx > 88) {
-				step_idx = 88;
-			}
+			step_idx = CLAMP(step_idx, 0, 88);
```

Comment:
This seemed too obvious an improvement *not* to do it, though it's out of scope of the rest of this PR's cosmetic changes.